login.css

```diff
@@ -161,6 +176,8 @@ p{
 	position: absolute;
   	top: 230px;
 	left:0px;
+	font-family: 'Roboto', sans-serif;
```

Comment:
13/08/2021: I set the fonts for the text in connect text to roboto and because it is normal text I set the font weight to 300. (remember that this .connected wasn't work for the login page because the  outcome were either display an error and go to home page)